src/newcharacter.cpp

```diff
@@ -1390,6 +1400,7 @@ tab_direction set_profession(WINDOW *w, player *u, points_left &points)
                                      pointsForProf);
         }
         // This string has fixed start pos(7 = 2(start) + 5(length of "(+%d)" and space))
+        int pMsg_length = utf8_width( points.to_string().c_str() );
```

Comment:
```utf8_width()``` accepts ```std::string``` as a parameter, so no need to use ```.c_str()``` here (and in other instances of this function as well).